quinn/examples/ack_timestamp_test.rs

```diff
@@ -0,0 +1,83 @@
+//! This example intends to use the smallest amount of code to make a simple QUIC connection.
+//!
+//! Checkout the `README.md` for guidance.
+
+use std::error::Error;
+use std::time::{Duration, Instant};
+
+mod common;
+use common::{make_client_endpoint, make_server_endpoint};
+
+use bytes::BufMut;
+
+use tracing::{self, info, trace, trace_span};
+use tracing_subscriber::{fmt, prelude::*, EnvFilter};
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
+    let test_length = Duration::from_secs(15);
+    // This should match approximately what the interpacket delay is.
+    let send_interval = Duration::from_millis(250);
+
+    tracing_subscriber::registry()
+        .with(fmt::layer())
+        .with(EnvFilter::from_default_env())
+        .init();
+
+    let server_addr = "127.0.0.1:20001".parse().unwrap();
```

Comment:
The `server_addr` and `client_endpoint` have ports that are impacted by the traffic shaping script.